### DISCUSSION TITLE
change media download preferences

### DISCUSSION THREAD

**Discussion by gardenkayak102** [aclap-dev/video-downloadhelper#1412] [Help and Q&A]
### Your question

Good day, I accidentally clicked MP4 as the "always prefer" this type of media.  I went to the settings and clicked the blue underlined link to try and change the preference to allow all (including, especially, video).  The link is faulty, and I didn't get any options. 

I re-installed (I have premium) and now it doesn't recognize any media at all.  I also entered my license key and when I click check license, it won't verify the license. 

Does anyone know how to fix these issues? 
Thanks! 

### Addon & CoApp version and details

Version 9.1.0.44 (stable), target mozilla, locale en-US.

**Comment by gardenkayak102** [comment]
Adding on that I was able to verify my license, but the media preferences "clear" button does not work, and I would still like to be able to download media.  I have tried pressing the play button on video, and still, its not showing since I accidentally clicked on always prefer MP4.  Your help is appreciated :)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You're right about the `Clear` button in the VDH settings not appearing to do anything.  But that setting only biases VDH to show that type first.  It doesn't restrict VDH to that type.  Here's a randomly chosen example:

![#01](https://github.com/user-attachments/assets/5962fbf3-589e-4ca3-9828-fb87ee32fb22)

Notice how the MP4 variants are listed first, obeying my setting.  But the MKV variants are offered & I could download one of them if I wanted.  Isn't this what you are seeing?  Maybe you ought to provide an example where you aren't getting all of that.  An example means NOT just the name of a web site.  Post the URL of a specific video on which you are not getting the full complement of offerings.

Also, when you provide information about your environment, do it like this:

Windows 11 Pro 64-bit version 24H2 build 26100.2161, Firefox 131.0.3 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20

There's 4 pieces of information there.  You should always provide all 4 of those.
